### PR TITLE
Feature/upvote button

### PR DESCRIPTION
**Pull request by donghee214**
- Import styles so it's visible in the scope of all stories
- Add votebutton & the animations involving it
- Fixed module resolvers for pathing inside storybook & src directories
- Add font-family css

### CODE REVIEW

File: src/assets/App.css [hunk 1]
```diff
@@ -12,10 +26,14 @@ body {
   text-align: center;
 }
 
-/* Import Baseline styles */
-
-@import url("./shared/colours.css");
```
Comment (DanialBetres) [inline src/assets/App.css]: why are you deleting this?

File: .storybook/stories/components/VoteButton.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+// import VoteButton from "shared/components/VoteButton";
+import VoteButton from "shared/components/VoteButton";
+export default class VoteButtonContainer extends PureComponent {
+  state = {
+    active: false
+  };
+
+  render() {
+    return (
+      <div style={{ display: "flex" }}>
```
Comment (DanialBetres) [inline .storybook/stories/components/VoteButton.js]: We should move these inline styles into classes 🌮

File: src/shared/components/VoteButton.tsx [hunk 3]
```diff
@@ -1,12 +1,47 @@
...
+  currentAnimation: string;
+}
+
+export default class extends PureComponent<Props, State> {
+  state = {
+    currentAnimation: ""
+  };
+
+  onClick = () => {
+    this.setState({ currentAnimation: "" });
```
Comment (DanialBetres) [inline src/shared/components/VoteButton.tsx]: Do you need to set state here?

File: src/shared/components/VoteButton.tsx [hunk 4]
```diff
@@ -1,12 +1,47 @@
...
+
+  render() {
+    return (
+      <button
+        onClick={this.onClick}
+        className={`cbutton cbutton--effect-ivana ${
+          this.state.currentAnimation
+        }`}
+      >
+        <svg
```
Comment (DanialBetres) [inline src/shared/components/VoteButton.tsx]: I think we should put this in it's own file...  and then we can do what you suggested which is to make it a react component with optional params to adjust colour, width, height, etc...

File: src/assets/js/shared/components/VoteButton.css [hunk 5]
```diff
@@ -1,5 +1,192 @@
...
+  }
+  50%,
+  100% {
+    opacity: 0;
+    -webkit-transform: scale3d(1.2, 1.2, 1);
+    transform: scale3d(1.2, 1.2, 1);
+  }
+}
+
+@-webkit-keyframes anim-effect-ivana-1-reverse {
```
Comment (DanialBetres) [inline src/assets/js/shared/components/VoteButton.css]: What is all of this 😖

File: src/shared/components/VoteButton.tsx (hunk 3 above)
Comment (donghee214) [reply src/shared/components/VoteButton.tsx]: Yeahhh, cause we're using css class animations, need to remove the class and re-add it to perform the animation

File: src/shared/components/VoteButton.tsx (hunk 4 above)
Comment (donghee214) [reply src/shared/components/VoteButton.tsx]: Agreed, kk

File: src/assets/js/shared/components/VoteButton.css (hunk 5 above)
Comment (donghee214) [reply src/assets/js/shared/components/VoteButton.css]: The coolest tap animation ever

**Pull request merged**